tests/adapter/dbt/tests/adapter/basic/files.py

```diff
@@ -201,7 +201,7 @@
 incremental_not_schema_change_sql = """
 {{ config(materialized="incremental", unique_key="user_id_current_time",on_schema_change="sync_all_columns") }}
 select
-    1 || '-' || current_timestamp as user_id_current_time,
+    {{ dbt.concat(["1", "'-'" , "current_timestamp"]) }} as user_id_current_time,
```

Comment:
I didn't actually fully examine this code, but I'm wondering if this would be even more compatible across different databases and platforms?

```suggestion
    {{ dbt.concat(["1", "'-'" , dbt.current_timestamp()]) }} as user_id_current_time,
```

Reply:
Correct :) Thanks!